### ISSUE TITLE
Met with empty schemas page after docker-compose upgrade

### ISSUE THREAD

**Issue by pavish** [mathesar-foundation/mathesar#3426]
## Description
* Performed a guided install of 0.1.3
  * It connects a single external user DB with 2 schemas and multiple tables.
* Upgraded to 0.1.4 following the information in the latest docs (from https://github.com/mathesar-foundation/mathesar/pull/3227).
  * Moved Postgres volumes as mentioned in docs.
* Upgrade succeeded.
* Met with the following page:
  <img width="716" alt="Screenshot 2024-01-29 at 7 17 17 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/eca09f3c-8024-4fd9-9b89-5ce2a87beb15">
* I repeated the upgrade twice and this happened both times.
* The database name, host, and port of the user database connection were empty.
  <img width="984" alt="Screenshot 2024-01-29 at 7 18 48 PM" src="https://github.com/mathesar-foundation/mathesar/assets/11850603/eefc7409-fedf-4663-8744-15563a889e03">

**Comment by mathemancer** [comment]
@pavish Can you give me a little detail about whether you started with `MATHESAR_DATABASES` (I assume yes), whether you removed it, and when you removed it?

**Comment by pavish** [comment]
@mathemancer I did not remove it.

* I followed the guided install script which installed Mathesar and set up the `MATHESAR_DATABASES` env variable.
  * I chose 'use existing' instead of 'create new' in the guided install DB prompt, and gave the credentials to an existing database.
* I started Mathesar, it worked fine.
* I then following the upgrade instructions as per the docs, and met with this error.
